src/Tokenizers/PHP.php

```diff
@@ -466,6 +475,7 @@ class PHP extends Tokenizer
         T_CLASS                    => true,
         T_INTERFACE                => true,
         T_TRAIT                    => true,
+        T_ENUM                     => true,
```

Comment:
I have a feeling that `T_ENUM_CASE` could also be added here as the first non-whitespace token after `T_ENUM_CASE` should always be tokenized as `T_STRING` or am I overlooking something ?

Might also be a good idea to cover that via the tests, i.e.:
```php
enum {
    /* testKeywordAsCaseNameShouldBeTstring1 */
    case FUNCTION = 'function';
    /* testKeywordAsCaseNameShouldBeTstring2 */
    case FALSE = 'false';
    /* testKeywordAsCaseNameShouldBeTstring3 */
    case DEFAULT = 'default'; // Should maybe also be added to the `DefaultKeywordTest` file ?
    /* testKeywordAsCaseNameShouldBeTstring4 */
    case ARRAY = 'array';
}
```

Reply:
Needs https://github.com/squizlabs/PHP_CodeSniffer/pull/3484